src/aiida/cmdline/commands/cmd_archive.py

```diff
@@ -521,6 +522,12 @@ def _import_archive_and_migrate(
                     )
             else:
                 _echo_exception(f'an exception occurred while trying to import the archive {archive}', exception)
+        except ImportTestRun:
+            echo.echo_success(
+                f'Import dry-run of archive {archive} terminated successfully. Profile storage unmodified.'
+            )
```

Comment:
Message will already be prefixed with `Success:` so `successfully` is a bit redundant
```suggestion
            echo.echo_success(f'Dry-run of archive `{archive}` completed; profile storage unmodified.')
```